### PR TITLE
[MRG] Fixed bugs of Kaggle integration

### PR DESCRIPTION
**Pull request by huangyz0918**

### CODE REVIEW

File: mle/agents/advisor.py [hunk 1]
```diff
@@ -102,11 +102,12 @@ def __init__(self, model, console=None):
         self.sys_prompt += self.json_mode_prompt
         self.chat_history.append({"role": 'system', "content": self.sys_prompt})
 
-    def suggest(self, requirement):
+    def suggest(self, requirement, process=True):
         """
         Handle the query from the model query response.
         Args:
             requirement: the user requirement.
+            process: whether to process the report.
```
Comment (HuaizhengZhang) [inline mle/agents/advisor.py:110]: sry. What does this process mean?
coud you give an e.g. in this function doc?
as `process the report` is too general for me

File: mle/agents/coder.py [hunk 2]
```diff
@@ -49,19 +52,37 @@ def __init__(self, model, working_dir='.', console=None):
...
-        7. Generate the commands to run and test the current task, and the dependencies list for this task.
-        8. You only write Python scripts, don't write Jupiter notebooks which require interactive execution.
+        - Writing the code into the file when creating new files, do not create empty files.
+        - Use function `preview_csv_data` to preview the CSV data if the task include CSV data processing.
+        - Decide whether the task requires execution and debugging before moving to the next or not.
+        - Generate the commands to run and test the current task, and the dependencies list for this task.
+        - You only write Python scripts, don't write Jupiter notebooks which require interactive execution.
         """
+
+        if single_file:
```
Comment (HuaizhengZhang) [inline mle/agents/coder.py:66]: can we add a `todo` here?

the 2 prompts here have many redundancies.

File: mle/cli.py [hunk 3]
```diff
@@ -113,29 +112,47 @@ def report(ctx, repo, model, user, visualize):
...
+@click.option('--sub_example', default=None, help='the path to the kaggle submission example .csv file.')
+@click.option('--comp_id', default=None, help='the kaggle competition id.')
+def kaggle(model, auto, description=None, datasets=None, sub_example=None, submission='.', comp_id=None):
     """
     kaggle: kaggle competition workflow.
+
+    Example command for auto mode:
+    mle kaggle --auto --datasets "/Users/huangyz0918/desktop/spaceship-titanic/prepared/public/train.csv,/Users/huangyz0918/desktop/spaceship-titanic/prepared/public/test.csv"
+     --description "/Users/huangyz0918/desktop/spaceship-titanic/prepared/public/description.md" --submission "./submission.csv"
+     --sub_example "/Users/huangyz0918/desktop/spaceship-titanic/prepared/public/sample_submission.csv" --comp_id "spaceship-titanic"
```
Comment (HuaizhengZhang) [inline mle/cli.py:128]: tremendous example

**Review by HuaizhengZhang — APPROVED** [review]

File: mle/cli.py (hunk 3 above)
Comment (HuaizhengZhang) [reply mle/cli.py:128]: please update the readme section also

otherwise, people can not enjoy this function

File: mle/agents/coder.py (hunk 2 above)
Comment (huangyz0918) [reply mle/agents/coder.py:66]: Sure

**Pull request merged**